easybuild/easyconfigs/n/n2v/n2v-0.3.2-foss-2022a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonBundle'
```

Comment:
I might have missed a memo :), are we using `PythonBundle` even for a single python package?

Reply:
Actually yes. I think we said it was less messing about in those cases where we need to add or remove deps, from just add/remove a few lines to completely restructure the easyconfig. But we haven't used PythonPackage as a reason to hold anything back from a merge.